src/baguetteBox.js

```diff
@@ -313,8 +327,19 @@
         if (options.buttons === 'auto' && ('ontouchstart' in window || imagesMap[currentGallery].length === 1)) {
```

Comment:
completely unrelated, but: am i right in thinking this test for `ontouchstart` is used to determine whether or not buttons should be shown? if so, this will fall foul of touchscreen-enabled laptops/desktops and mobile/tablet devices with paired mouse/keyboard - see http://patrickhlauke.github.io/getting-touchy-presentation/